specs/PointOfInterest/WeatherStation/generate.py

```diff
@@ -0,0 +1,255 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+
```

Comment:
we need requirements.txt for this program

Reply:
Fixed in https://github.com/FIWARE/dataModels/pull/386/commits/939738767c0ec7f2e36bf9376fc3014e0fffc0e7